packages/mock/src/client.ts

```diff
@@ -154,9 +156,11 @@ export class MockClient extends MedplumClient {
         tableLayouts?: { [name: string]: CustomTableLayout },
         fonts?: TFontDictionary
       ) => client.mockCreatePdf(docDefinition, tableLayouts, fonts),
-      fetch: (url: string, options: any) => {
-        return client.mockFetch(url, options);
-      },
+      fetch: clientOptions?.fetch
+        ? clientOptions.fetch
+        : (url: string, options: any) => {
+            return client.mockFetch(url, options);
+          },
     });
```

Comment:
Super minor, but I think you should be able to use `clientOptions?.fetch ?? ...`

Reply:
Nope, I tried that. You can't do nullish-coallescing if the right side is an arrow function. Looks like it creates a syntax error:
`Uncaught SyntaxError: Malformed arrow function parameter list`